demos/testing/test_database_backed_server/test_app.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from tornado import testing
+from app import create_app
+
+class SimpleTest(testing.AsyncHTTPTestCase):
+
+    def get_app(self):
+        application, ioloop = create_app()
+        return application
+
+    def get_new_ioloop(self):
+        application, ioloop = create_app()
```

Comment:
This is incorrect - if `create_app` created a new IOLoop instead of just using `instance()`, you'd have an app created in `get_app` that used a different IOLoop than the one in `get_new_ioloop`. Fortunately, you don't need to override `get_new_ioloop` at all in this case. Just let `get_app` use the `current()` IOLoop that's created for you.